### ISSUE TITLE
Supported Datatype for Fine-Tuning/Retraining 

### ISSUE THREAD

**Issue by laxmimerit** [open-mmlab/mmsegmentation#1818]
Hi,
I am trying to use `segformer` model on custom dataset. I have used `Label Studio` for my data annotation. I get following options to export the annotion.
![image](https://user-images.githubusercontent.com/16714612/180769783-b6f89bb3-052d-4e61-8ebc-4c37a4550f0b.png)

I want to use `Coco` format but couldn't find any tutorial. Please suggest me any link to use `Coco` format on custom dataset.

**Comment by MengzhangLI** [comment]
Sorry I have no idea on `Label Studio` because I never used it before. Perhaps this dataset paration may help you: https://github.com/open-mmlab/mmsegmentation/blob/master/docs/en/dataset_prepare.md#coco-stuff-164k.

**Comment by laxmimerit** [comment]
Thanks @MengzhangLI for a quick response. I have downloaded annotation in `Coco` format. Can you please share finetuning document using Coco dataset?

**Comment by MengzhangLI** [comment]
> Thanks @MengzhangLI for a quick response. I have downloaded annotation in `Coco` format. Can you please share finetuning document using Coco dataset?

No, we don't have certain finetuning document using Coco dataset.
